common/pulp_docker/common/models.py

```diff
@@ -15,7 +15,7 @@ class Blob(object):
     """
     This class is used to represent Docker v2 blobs.
     """
-    TYPE_ID = 'docker_blob'
+    TYPE_ID = constants.BLOB_TYPE_ID
```

Comment:
This change will conflict with the master branch because I've removed these there.